watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/TaskAbstractFragment.java

```diff
@@ -383,13 +393,29 @@ public void onClick(DialogInterface dialogInterface, int i) {
 
         }
 
-        public String[] getUserNames() {
-            String[] names = new String[mUsers.size()];
-            for (int i = 0; i < mUsers.size(); i++) names[i] = mUsers.get(i).getName();
-            return names;
+        public List<User> getUsers() {
+            List<User> userList = new ArrayList<>();
+            for (User user : mUsers) {
+                if (user.getRoleString().equals(User.MANAGER)) mAdmin.add(user);
+                else if (user.getRoleString().equals(User.EMPLOYEE)) mEmployee.add(user);
+                else if (user.getRoleString().equals(User.COMMUNITY_MEMBER)) mMember.add(user);
+            }
+            User mAdminHeader = new User(User.MANAGER);
+            User mEmployeeHeader = new User(User.EMPLOYEE);
+            User mMemberHeader = new User(User.COMMUNITY_MEMBER);
+
+            userList.add(mMemberHeader);
```

Comment:
Do these just appear as null users? and what happens if a user clicks on one of them?